### PR TITLE
Jenkin changes to support tier-1 4.x cephfs

### PR DESCRIPTION
**Pull request by anrao19**
# Description
Signed-off-by: Anuchaithra Rao <anrao@redhat.com>

Jenkin changes to support tier-1 4 cephfs
https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/view/RHCS%204%20QE/job/rhceph-4-tier-1-cephfs/

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: pipeline/4/Jenkinsfile-tier-1-cephfs.groovy [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                userRemoteConfigs: [[
+                    url: 'https://github.com/red-hat-storage/cephci.git'
+                ]]
+            ])
+            sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+            sharedLib.prepareNode()
+        }
+    }
+
+    timeout(unit: "MINUTES", time: 360) {
```
Comment (psathyan) [inline pipeline/4/Jenkinsfile-tier-1-cephfs.groovy:38]: The timeout needs to be controlled from the wrapper job... not individual ones... Would it be possible to remove this tag from here?

Comment (anrao19) [reply pipeline/4/Jenkinsfile-tier-1-cephfs.groovy]: Sure, will do that

File: pipeline/4/Jenkinsfile-tier-1-wrapper.groovy [hunk 2]
```diff
@@ -61,8 +62,8 @@ node(nodeName) {
 		println "Triggering tier-1 jobs for compose ${composeInfo}"
 	}
 
-	timeout(unit: "HOURS", time: 24) {
-        for(jobName in tier1Jobs) {
+	for(jobName in tier1Jobs) {
+        timeout(unit: "HOURS", time: 5) {
```
Comment (psathyan) [inline pipeline/4/Jenkinsfile-tier-1-wrapper.groovy:66]: IMO, let's limit the runtime execution to 24 hours and not cap the individual jobs... What do you think?

Comment (anrao19) [reply pipeline/4/Jenkinsfile-tier-1-wrapper.groovy:66]: Sure

**Review by psathyan — APPROVED** [review]

**Pull request merged**